.github/CODEOWNERS

```diff
@@ -146,36 +143,24 @@
 
 
 # Dev workflow team.
-# TODO(#11083): Re-add owner @BenHenning after 2020-11-15.
-/.circleci/ @U8NWXD @kevintab95 @vinitamurthi
-# TODO(#10619): Add @bansalnitish as owner after 2020-10-31
```

Comment:
Doesn't seem like Nitish is being added back--is that intentional?

Reply:
Yes, that is correct. Srijan now owns the relevant files as lead of the Angular migration team.